src/mappings/projectRegistry.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020-2022 SubProject Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  CreateProjectEvent,
+  ServiceStatusChangedEvent,
+  TransferEvent,
+  UpdateProjectDeploymentEvent,
+  UpdateProjectMetadataEvent,
+} from '@subql/contract-sdk/typechain/ProjectRegistry';
+import { EthereumLog } from '@subql/types-ethereum';
+import assert from 'assert';
+import {
+  Deployment,
+  IndexerDeployment,
+  Project,
+  ProjectType,
+  ServiceStatus,
+} from '../types';
+import { biToDate, bytesToIpfsCid } from './utils';
+
+function getIndexerDeploymentId(indexer: string, deploymentId: string): string {
+  return `${indexer}:${deploymentId}`;
+}
+
+export async function handleNewProject(
+  event: EthereumLog<CreateProjectEvent['args']>
+): Promise<void> {
+  logger.info('handleNewProjectProject');
+  assert(event.args, 'No event args');
+
+  const {
+    creator,
+    projectId,
+    projectMetadata,
+    projectType,
+    deploymentId,
+    deploymentMetadata,
+  } = event.args;
+  const type = projectType as unknown as ProjectType;
+
+  const project = Project.create({
+    id: projectId.toHexString(),
+    owner: creator,
+    type,
+    metadata: projectMetadata,
```

Comment:
remove `ipfs://` prefix, or other places need to adapt to the difference.

Reply:
`projectMetadata` doesn't include `ipfs://`, only user call `tokenURI` will contain the base url